springwolf-core/src/test/java/io/github/springwolf/core/integrationtests/AsyncApiDocumentWithOpenApiV31SchemaFormatIntegrationTest.java

```diff
@@ -0,0 +1,437 @@
+// SPDX-License-Identifier: Apache-2.0
+package io.github.springwolf.core.integrationtests;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.github.springwolf.asyncapi.v3.jackson.AsyncApiSerializerService;
+import io.github.springwolf.asyncapi.v3.model.AsyncAPI;
+import io.github.springwolf.asyncapi.v3.model.channel.message.Message;
+import io.github.springwolf.asyncapi.v3.model.channel.message.MessageObject;
+import io.github.springwolf.asyncapi.v3.model.components.ComponentSchema;
+import io.github.springwolf.asyncapi.v3.model.schema.SchemaFormat;
+import io.github.springwolf.asyncapi.v3.model.schema.SchemaReference;
+import io.github.springwolf.core.asyncapi.AsyncApiService;
+import io.github.springwolf.core.asyncapi.scanners.common.headers.AsyncHeadersNotDocumented;
+import io.github.springwolf.core.fixtures.MinimalIntegrationTestContextConfiguration;
+import io.github.springwolf.core.integrationtests.application.fqn.FqnApplication;
+import io.github.springwolf.core.integrationtests.application.listener.ListenerApplication;
+import io.github.springwolf.core.integrationtests.application.polymorphic.PolymorphicPayloadApplication;
+import io.github.springwolf.core.integrationtests.application.publisher.PublisherApplication;
+import io.github.springwolf.core.integrationtests.application.schema.SchemaEnumAsRefApplication;
+import io.github.springwolf.core.standalone.DefaultStandaloneApplication;
+import io.swagger.v3.oas.models.media.Schema;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.springframework.test.context.TestPropertySource;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AsyncApiDocumentWithOpenApiV31SchemaFormatIntegrationTest {
```

Comment:
review: do we need all the tests on this level?
I wonder if one happy path test case is sufficient. Feels like a lot of test duplication

Reply:
I see, still it feels like we are testing an external library.

Let's do the following: Include the updated tests as you suggest.
At the same time, https://github.com/springwolf/springwolf-core/pull/1509 is an alternative way to run these core integration tests (I agree that currently, there are no easy to use integration tests in core).

After this PR is merged, we / I re-visit this.